bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImpl.java

```diff
@@ -175,8 +176,12 @@ protected void initModel() {
                         boolean isWCMDisabled =  (com.day.cq.wcm.api.WCMMode.fromRequest(request) == com.day.cq.wcm.api.WCMMode.DISABLED);
                         String dmServerUrl;
                         if (!isWCMDisabled) {
-                            //for Author
-                            dmServerUrl = "/is/image/";
+                            if (asset.getMetadataValue(Scene7Constants.PN_S7_TYPE).equals(Scene7AssetType.ANIMATED_GIF.getValue())) {
```

Comment:
Please import the constants from `DMAssetPostProcessor.java`!

Reply:
hello there. i am trying to add your suggested change, but DMAssetPostProcessor only has two (private!) constants: IMAGE_SERVER_PATH and CONTENT_SERVER_PATH
i could make them public and use them here. is that what you are referring to?

Reply:
Yes, please reuse them!

Reply:
OK - should be done now. shall i resolve this conversation?